internal/rbd/migration.go

```diff
@@ -77,35 +77,43 @@ func parseMigrationVolID(vh string) (*migrationVolID, error) {
 // deleteMigratedVolume get rbd volume details from the migration volID
 // and delete the volume from the cluster, return err if there was an error on the process.
 func deleteMigratedVolume(ctx context.Context, parsedMigHandle *migrationVolID, cr *util.Credentials) error {
+	rv, err := genVolFromMigVolID(ctx, parsedMigHandle, cr)
+	if err != nil {
+		return err
+	}
```

Comment:
@humblec have you tested these changes manually?

Reply:
@Madhu-1 Thats what in the PR description?

Reply:
Have you tested the delete workflow the rv connect and destroy is done in genVolFromMigVolID but the connection is needed for deleteImage to work

Reply:
This change was not part of , actually I did that as an optimization. I can recheck again..

Reply:
yeah,  reverted it..